FuzzySharp/PreProcess/StringPreprocessorFactory.cs

```diff
@@ -5,7 +5,7 @@ namespace FuzzySharp.PreProcess
 {
     internal class StringPreprocessorFactory
     {
-        private static string pattern = "[^ a-zA-Z0-9]";
+        private static string pattern = "[^ a-zA-Z0-9а-зА-З]";
```

Comment:
This is interesting and good initiative.
I think a better way to do this is by updating the `PreprocessMode` enum to accept, the enum is confusing and `Full` vs `None` does not make much sense.\
Also flags makes sense in case I'm working with more than one language.
I propose the following:
```cashrp
[Flags]
public enum PreprocessMode
    {
        NotSet = 0,
        English = 1,
        Russian = 2,
        Gibberish = 5 
    }
```
Then here, in this method use the correct pattern(s).
If `PreprocessMode==1` then `pattern = "[^ a-zA-Z0-9]";` // English
If `PreprocessMode==2` then `pattern = "[^а-зА-З0-9]";` // Russian
If `PreprocessMode==5` then `pattern = "[^a-zA-Z0-9а-зА-З]";` //Both English & Russian

Finally, even the name `PreprocessMode` isn't very descriptive, maybe `FuzzyLanguageProcessor` or something like that would be a better name.